sympy/polys/domains/pythonrational.py

```diff
@@ -42,7 +42,8 @@ def parent(self):
         from sympy.polys.domains import PythonRationalField
         return PythonRationalField()
 
-    def __init__(self, p, q=1):
+    def __init__(self, p, q=1, _gcd=True):
+        from sympy.polys.domains.groundtypes import python_gcd as gcd
```

Comment:
The imports have to go inside the functions because of circular import issues (groundtypes imports this file).